01-map/transformSecondsToWords.js

```diff
@@ -5,3 +5,9 @@
  */
 
 const SECONDS = [2, 5, 100];
+
+function transformSecondsToWords(seconds) {
+  return seconds.map(String);
```

Comment:
Can you explain how this callback is working? Why are you using only `String` as callback (with no arguments) and why are you using `String`? 🤔

I wait for your motivations regarding this.